### ISSUE TITLE
logger 8.0.16

### ISSUE THREAD

**Issue by Krakozaber** [mpdf/mpdf#1604]
mpdf v8.0.16

When upgrading from version 8.0.15 to version 8.0.16. The logger has not been changed.

Declaration of Mpdf\Mpdf::setLogger(Psr\Log\LoggerInterface $logger) must be compatible with Psr\Log\LoggerAwareInterface::setLogger(Psr\Log\LoggerInterface $logger): void

**Comment by finwe** [comment]
Yes, I was too hasty. Will revert.

**Comment by finwe** [comment]
Fixed in v8.0.17

**Comment by dillingham** [comment]
@finwe I can't update to 8.0.17 because of:

```
Problem 1
    - Root composer.json requires mpdf/mpdf 8.0.17 -> satisfiable by mpdf/mpdf[v8.0.17].
    - mpdf/mpdf v8.0.17 requires psr/log ^1.0 || ^2.0 -> found psr/log[1.0.0, ..., 1.1.4, 2.0.0] but the package is fixed to 3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

**Comment by finwe** [comment]
@dillingham 
1) you are off topic here 
2) yes, you can https://stackoverflow.com/questions/74433569/mpdf-mpdf-loggerawareinterface-incompatibility-with-psr-log-in-php-8-1-12/74442440#74442440
